packages/caravan-fees/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@caravan/fees",
+  "version": "0.0.0",
```

Comment:
Can i suggest we make this `0.0.0-beta`?

Reply:
Done [here](https://github.com/caravan-bitcoin/caravan/commit/4fce4fe2963337a6dcf56c153b80ab4fab1a5aa4#diff-a3e31745046775327f41608ad7eccffc05717d319e1f0fb67360040d4488fd61R3)